bin/pa11y.js

```diff
@@ -244,3 +250,11 @@ function outputEnvironmentInfo() {
 	console.log('PhantomJS:  ' + versions.phantom);
 	console.log('OS:         ' + versions.os + ' (' + process.platform + ')');
 }
+
+function checkPackageVersion(program) {
+	if (!program.doNotCheckPackageVersion) {
```

Comment:
Hey @andygnewman, shouldn't this be `program.noPackageVersionCheck` to correlate to the new `--no-package-version-check` flag?  Looks like a little naming mismatch.